# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/PostTraversalTask.h
# Commit: 93b7e171c74a
# Full Hash: 93b7e171c74a6aa88d98979009eb0e299e2b61a3
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 1. Split FontFace(Set) into DOM facing and implementation classes. r=emilio
#   
#   This patch aims to split the implementation details of FontFace(Set)
#   into FontFace(Set)Impl classes. The Impl variants have threadsafe
#   refcounting and are intended to be the basis for worker support for
# ==============================================================================

diff -r c45dacd63805 -r 93b7e171c74a layout/style/PostTraversalTask.h
--- a/layout/style/PostTraversalTask.h	Tue Jun 28 14:12:34 2022 +0000
+++ b/layout/style/PostTraversalTask.h	Tue Jun 28 14:27:20 2022 +0000
@@ -16,6 +16,7 @@
 namespace dom {
 class FontFace;
 class FontFaceSet;
+class FontFaceSetImpl;
 }  // namespace dom
 namespace fontlist {
 struct Family;
@@ -59,7 +60,7 @@
   }
 
   static PostTraversalTask DispatchFontFaceSetCheckLoadingFinishedAfterDelay(
-      dom::FontFaceSet* aFontFaceSet) {
+      dom::FontFaceSetImpl* aFontFaceSet) {
     auto task = PostTraversalTask(
         Type::DispatchFontFaceSetCheckLoadingFinishedAfterDelay);
     task.mTarget = aFontFaceSet;
@@ -101,7 +102,7 @@
     // mTarget (FontFaceSet*)
     DispatchLoadingEventAndReplaceReadyPromise,
 
-    // mTarget (FontFaceSet*)
+    // mTarget (FontFaceSetImpl*)
     DispatchFontFaceSetCheckLoadingFinishedAfterDelay,
 
     // mTarget (gfxUserFontEntry*)